howto/android/custom-vhal.md

```diff
@@ -136,6 +136,100 @@ addons:
   - custom-vhal
 ```
 
+## Integrate Anbox VHAL interface
+
+To use and integrate the [Anbox VHAL interface](https://github.com/canonical/vendor_canonical_interfaces/tree/main/vehicle) into a VHAL implementation, the implementation must be built against Android 14. Before proceeding, ensure you have downloaded the Android 14 source. If you haven't done so yet, please follow the [official documentation](https://source.android.com/docs/setup/download) to set it up.
+
+1. Add a new remote definition named `github` to `.repo/manifests/manifest.xml` file.
+
+       <remote name="aosp"
+          fetch=".."
+          review="https://android-review.googlesource.com/" />
+
+       <remote name="github"
+          fetch="https://github.com/canonical/" />
+
```

Comment:
Nitpick: could we add the language to have syntactic coloration?
````suggestion
```xml
       <remote name="aosp"
          fetch=".."
          review="https://android-review.googlesource.com/" />

       <remote name="github"
          fetch="https://github.com/canonical/" />
```
````

And same for the ones below.